products/opensuse/main.pm

```diff
@@ -38,6 +38,10 @@ sub is_leap {
     return get_var('VERSION') =~ /^42:S/;
 }
 
+sub is_leap_42_3_or_newer {
+    return is_leap && check_var('VERSION', '42.3');
+}
+
```

Comment:
Alright, now we have leap_version_at_least in utils, which will force test dev to extend function if the use it for something besides 42.2 and 42.3.